packages/Webkul/Velocity/src/Resources/views/shop/customers/session/index.blade.php

```diff
@@ -72,10 +72,10 @@ class="form-style"
                                     v-validate="'required'"
                                     value="{{ old('password') }}"
                                     data-vv-as="&quot;{{ __('shop::app.customer.login-form.password') }}&quot;" />
-                                    <input type="checkbox" onclick="myFunction()">Show Password
+                                <input type="checkbox" onclick="myFunction()" id="shoPassword" style="margin-top:10px;">Show Password
```

Comment:
- Please remove the static text Show Password and use translations
- Please remove the style tag and place the css code in the scss files & recompile the assets